### DISCUSSION TITLE
Will my links become broken if I upgrade my version?

### DISCUSSION THREAD

**Discussion by RafaelSPBarbosa** [filebrowser/filebrowser#1665] [Q&A] [answered]
Hey,

I've used some of the sharing links with some partners and clients. I saw that a lot of new features and fixes came out since I first installed filebrowser. I'd like to know if these links would become broken if I were to update my filebrowser version. I'm not running docker or anything special, just an ubuntu with nginx.

**Comment by ramiresviana** [comment, marked as answer]
The v2.18.0 has an issue with the share view, making the file list empty, and should be fixed on the next release. Except for that, you should have no problems when updating.
